pkgs/applications/audio/butt/default.nix

```
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchurl, pkg-config, fltk13, portaudio, lame, libvorbis, libogg
+, flac, libopus, libsamplerate, fdk_aac, dbus, openssl, curl }:
```

One per line.
Almost-asciibetical:

```suggestion
{ lib
, stdenv
, fetchurl
, curl
, dbus
, fdk_aac
, flac
, fltk13
, lame
, libogg
, libopus
, libsamplerate
, libvorbis
, openssl
, pkg-config
, portaudio
}:
```